src/img_hash.h

```diff
@@ -0,0 +1,29 @@
+#ifndef _OPENCV3_IMG_HASH_H_
```

Comment:
well, it not necessary to follow the headers defination of gocv, you can completely add your ow